introduction.md

```diff
@@ -1,2 +1,6 @@
 My name is Yuri Gaidarji and I`m beginning to study github at code school.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?